lib/last_frame.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require './lib/frame'
+
+class LastFrame < Frame
```

Comment:
Interesting use of a sub-class, impressive! (Although I'm finding it quite hard to follow exactly what's happening!)